### ISSUE TITLE
SELECT, syntax error not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#3046]
```abap
DATA l_temseid TYPE apql-temseid.
DATA l_id TYPE apql-qid.
SELECT temseid
  INTO l_temseid FROM apql
    ORDER BY PRIMARY KEY
    WHERE qid = l_id.
ENDSELECT.
```
